### PR TITLE
Adding community profile: leighphan.md

### PR DESCRIPTION
**Pull request by leighphan**

### CODE REVIEW

File: community/leighphan.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Leigh Phan
+
+Hi! 👋 I'm Leigh and I like designing, building community, and learning to code. My nephew Cory the corgi is more famous than you.
+
+- Twitter: [@leighphan](https://www.twitter.com/leighphan)
+- Website: [hello](http://leighphan.github.io/hello)
```
Comment (gr2m) [inline community/leighphan.md]: I think you can just set the URLs instead of using the markdown syntax, it will be easier to parse later with JavaScript

**Review by gr2m — CHANGES_REQUESTED** [review]

File: community/leighphan.md (hunk 1 above)
Comment (leighphan) [reply community/leighphan.md]: Good catch; I've removed the markdown formatting.

**Review by gr2m — APPROVED** [review]
👌

**Comment by machikoyasuda** [comment]
corgi-treadmill
For Cory <3

**Pull request merged**